priv/templates/nimble_template/.github/workflows/test.yml.eex

```diff
@@ -88,6 +88,15 @@ jobs:
       - name: Run codebase check
         run: mix codebase
 
+      - name: Ensure that localization files (POs, POTs) are up-to-date.
+        run:
+          mix gettext.extract-and-merge;
+
+          if [ -n "$(git diff --exit-code priv/gettext/)" ]; then
+            echo "The localization files (POs, POTs) are NOT up-to-date, run \"mix gettext.extract-and-merge\" on your local and push again.";
```

Comment:
can we have this condition in the `mix codebase` @byhbt so the developer can check it on local 🤔

also for the message, if we add the `ettext.extract-and-merge` into `codebase.fix` we can adjust the message here to `The localization files (POs, POTs) are NOT up-to-date, run \"mix codebase.fix`

Reply:
hi @andyduong1920 

I have one problem with this, because the `codebase.fix` command is injected via `NimbleTemplate.Addons.TestEnv`, if we include the `gettext.extract-and-merge` command to `codebase.fix`, then this add-on will depend on `TestEnv`. 

So if we want to add `gettext.extract-and-merge` to `codebase.fix` we need to accept the depend on `TestEnv`, to make sure that the `codebase.fix` is existed before we inject this command. 

Here is the screenshot of `mix.exs` content after apply `Gettext` addon only.

![Screen Shot 2022-01-06 at 2 53 01 PM](https://user-images.githubusercontent.com/948856/148348406-f66b439a-e8cb-46a5-a6e9-e16323cbb851.png)

Reply:
Hi @byhbt you can check the `@describetag required_addons: [:TestEnv]` in the test codebase :)